packages/slice-machine/src/pages/changes.tsx

```diff
@@ -111,7 +113,7 @@ const Changes: React.FunctionComponent = () => {
       return <AuthErrorPage authStatus={authStatus} />;
     }
     if (numberOfChanges === 0) {
-      return <NoChangesBlankSlate />;
+      return <NoChangesBlankSlate isPostPush={isPushed} />;
```

Comment:
How does this behave when I reload the page? I wonder if there's a way of knowing if we have ever pushed changes before. Otherwise, we only have local state. When we reload the page, we'll have 0 changes but no post push, and we'll get a different blank slate content.

Reply:
when we reload we see the content confirming there are no new changes to pushed (it's named prePush, but i just realised this names make no sense, will change it).
the postPush should be visible only right after the successful push.